### ISSUE TITLE
Funding Page: Total Fee amount should not be displayed as "Locked" in Claiming Box

### ISSUE THREAD

**Issue by BartuCulha** [PrimeDAO/prime-deals-dapp#787]
Steps to reproduce:
1. Connect wallet: 0xE834627cDE2dC8F55Fe4a26741D3e91527A8a498
2. Navigate here: https://prime-deals-staging.vercel.app/funding/1W83N13zHcyB18c7Eq7JHd

Currently:
We are displaying fees as locked
![Funding-Prime-Deals (10).png](https://images.zenhubusercontent.com/609a9f5f0a40bdfde66fe338/b1e43214-4480-4969-b98e-652ec73ce33a)

Expected:
Locked should be 0 otherwise it might confuse users.

**Comment by BartuCulha** [comment]
An update: After Claiming all tokens: Locked column will become 0

See here #788

**Comment by dkent600** [comment]
Locked should be zero and I think we should add a "Fee" column so it all adds up to the total  @BartuCulha  @blink-km

**Comment by vipergtsrz** [comment]
@dkent600 So can you confirm the final requirements for this? From what I understand, the locked column should stay the same and we're just adding a new fee column which will be 0 until all tokens are claimed and then it will be .3% of the total tokens?

**Comment by vipergtsrz** [comment]
@dkent600 Yes, that is how it currently works. So I just have to add a fee column right? And that fee calculation is 0.3% of the total tokens right?

**Comment by BartuCulha** [comment]
Yes numbers that are locked = .003 x Vested Tokens
